src/pages/app-contact/app-contact.tsx

```diff
@@ -159,11 +152,14 @@ export class AppContact {
     form.scrollIntoView({ block: 'start', behavior: 'smooth' });
   }
 
+  private className =
+    localStorage.getItem('isSafari') === 'false' ? 'webp' : 'hero';
```

Comment:
Why in this one the property to check is `isSafari` and in other ones this other `allowWebp`?